treeherder/intermittents_commenter/tasks.py

```diff
@@ -3,7 +3,7 @@
 from treeherder.intermittents_commenter.commenter import Commenter
 
 
-@task(name='intermittents-commenter', soft_time_limit=30 * 60, time_limit=31 * 60)
+@task(name='intermittents-commenter', soft_time_limit=75 * 60, time_limit=76 * 60)
```

Comment:
I'd be ok with increasing this up to say 2 hours to add some headroom (it's mainly there to prevent runaway tasks taking hours/days).

Reply:
Ok, will do!